includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -556,7 +552,8 @@ public function remote_get( $args = array() ) {
 			if ( empty( $post ) ) {
 				$formatted_post = false;
 			} else {
-				$formatted_post = Utils\prepare_post( $post );
+				$dt_post        = new DistributorPost( $post );
+				$formatted_post = $dt_post->to_insert( $new_post_args );
```

Comment:
This causes a BC change on this line below. 

https://github.com/10up/distributor/blob/aea7570ec2cf0460726ac4f9c54959151a078590/includes/classes/InternalConnections/NetworkSiteConnection.php#L562

The `$formatted_post` was passed as an object in the past but is now passed as an array.

However, the filter is conflicted anyways as [demonstrated by this code](https://github.com/10up/distributor/blob/d61bc74134051986f81a5ec4e0dfb85a72f505df/includes/classes/ExternalConnections/WordPressExternalConnection.php#L356-L370), it can sometimes be an object, sometimes an array.